intel-hex.js

```diff
@@ -817,6 +817,40 @@ class MemoryMap {
         return memMap;
     }
 
+
+    /**
+     * Returns a new instance of <tt>MemoryMap</tt>, containing only data between
+     * <tt>offset</tt> and <tt>offset</tt> + <tt>length</tt>.
+     *
+     * <br/>
+     * The returned <tt>MemoryMap</tt> might be empty.
+     * <br/>
+     * Internally, this uses <tt>subarray</tt>, so new memory is not allocated.
+     *
+     * @param {Number} offset The start offset
```

Comment:
When reading this at first it was unclear what the offset was referring to, address or index. Might just be me though.

Reply:
Yeah, it might be a bit clearer by changing this into "address"; none of the other docstrings around use the "offset" nomenclature.